src/components/v5/common/TableWithMeatballMenu/utils.tsx

```diff
@@ -12,6 +12,11 @@ export const makeMenuColumn = <T,>(
     cell: ({ row }) => {
       const props = getMenuProps(row);
 
-      return props ? <MeatBallMenu {...props} /> : undefined;
+      return props ? (
+        <MeatBallMenu
+          className="absolute top-4 right-4 md:top-auto md:right-auto"
```

Comment:
Is absolute needed on desktop?

Reply:
There is `md:relative` added in MeatBallMenu component

Reply:
So maybe you should add a wrapper here that will have absolute-related classes. This way you will separate the component's logic from the place where it's used